static/templates/subscription_table_body.handlebars

```diff
@@ -44,3 +44,30 @@
         </div>
     </div>
 </div>
+<div id="stream_privacy_modal" class="modal hide" tabindex="-1" role="dialog" aria-labelledby="stream_privacy_modal_label" aria-hidden="true">
+    <div class="modal-header">
+        <button type="button" class="close close-privacy-modal" aria-hidden="true">×</button>
+        <h3 id="stream_privacy_modal_label">{{t "Change Stream Privacy" }} <span class="email"></span></h3>
+    </div>
+    <div class="modal-body">
+        <p id="public-stream">
```

Comment:
(We'll also want to make selectors accessing them use e.g. `#stream_privacy_modal .public_stream` for clarity,